network/one.yaml

```diff
@@ -1,14 +1,13 @@
 ---
-block_worker: http://one.devnet-0chain.net/dns
```

Comment:
please, revert it.
the file is called "one.yaml". it's supposed, i guess, the one-network settings should be there. 
for six-network settings could be put in six.yaml. and probably six.yaml should be on ignored list, not keep it locally only.